task/tkn-bundle/0.2/tkn-bundle.yaml

```diff
@@ -172,6 +172,38 @@ spec:
           printf '"%s"' "$escaped_arg"
         }
 
+        # Check if task is kustomized
+        # A task considered to be kustomized when a kustomization file is found in the task dir
+        is_kustomized_task() {
+            local -r task_dir=$1
+            local -r kustomiztion_file="$task_dir/kustomization.yaml"
```

Comment:
For the task itself we mention that it finds both yaml and yml files, but for the kusotmization files here we only support one extension. I think it's worth adding a check for both `$task_dir/kustomization.yaml` and `$task_dir/kustomization.yml` to avoid potential misunderstandings.

(and also changing the echo command later -- maybe it doesn't have to include the extension at all there)

Reply:
I added a check for `kustomization.yaml` and `kustomization.yml` and updated the echo command